roles/sssd/templates/ldap_backup.sh.j2

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+_ldap_credentials="/etc/openldap/readonly-ldapsearch-credentials.bash"
+source ${_ldap_credentials}
+
+_ldap_dest="/root/ldap_backup"
+test -e "${_ldap_credentials}" || exit 1
+
+_daily_keep=31     # one month worth
+_weekly_keep=52    # one year worth
+
+for _each_ldap in "${domain_names[@]}"; do
+   echo "> running backup for ${_each_ldap} <"
+   test -e "${_ldap_dest}/${_each_ldap}" || mkdir "${_ldap_dest}/${_each_ldap}"
```

Comment:
```suggestion
   test -e "${_ldap_dest}/${_each_ldap}" || mkdir -p -m 700 "${_ldap_dest}/${_each_ldap}"
```

Reply:
The default permission values of new folders/files are 755/644. But since they are inside the `/root` folder, there is no point adding the `700` - it improves nothing. Only user that can access `/root` folder, and therefore also subfolders, is `root` acount.